conf.yaml

```diff
@@ -95,6 +95,7 @@ contents:
             current:    7.4
             index:      docs/en/getting-started/index.asciidoc
             branches:   [ master, 7.x, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3 ]
+            live:       *stacklive
```

Comment:
Does this setting mean that this book uses the same values as are specified in &stacklive for the Installation and Upgrade Guide?  If so, that's a lot easier than copying the same values over and over.

Reply:
It does.